sklearn/tests/test_public_functions.py

```diff
@@ -162,6 +162,7 @@ def _check_function_param_validation(
     "sklearn.model_selection.train_test_split",
     "sklearn.random_projection.johnson_lindenstrauss_min_dim",
     "sklearn.svm.l1_min_c",
+    "sklearn.linear_model.orthogonal_mp",
```

Comment:
Please move it to preserve alphabetical order.

Reply:
moved to preserve alphabetical order